src/frontend/komponenter/Fagsak/Søknad/Barna.tsx

```diff
@@ -73,6 +71,23 @@ const Barna: React.FunctionComponent = () => {
               )
             : [];
 
+    const oppdaterBarnMedMerketStatus = (barnaSomErSjekketAv: string[]) => {
+        skjema.felter.barnaMedOpplysninger.validerOgSettFelt(
+            skjema.felter.barnaMedOpplysninger.verdi.map(
+                (barnMedOpplysninger: IBarnMedOpplysninger) =>
+                    barnaSomErSjekketAv.includes(barnMedOpplysninger.ident)
+                        ? {
+                              ...barnMedOpplysninger,
+                              merket: true,
+                          }
+                        : {
+                              ...barnMedOpplysninger,
+                              merket: false,
+                          }
+            )
```

Comment:
Dette ble ganske tungvindt - ønsker egentlig å kun lage objekt 1 gang og heller kalle includes på linjen hvor det står "merket". Men fikk ikke til at map-funksjonen returnerte et nytt objekt uten at jeg brukte ternary 🤔

Reply:
skal sjekke det litt ut!

Reply:
Hva med dette
```js
skjema.felter.barnaMedOpplysninger.verdi.map(
                (barnMedOpplysninger: IBarnMedOpplysninger) => ({
                    ...barnMedOpplysninger,
                    merket: barnaSomErSjekketAv.includes(barnMedOpplysninger.ident),
                })
            )
```

Reply:
Takk, det funket!! Prøvde aldri med paranteser rundt 😅

Reply:
Med paranteser rundt krøllklammene så sier du at krøllklammene skal tolkes som en returverdi i stedet for som en funksjonskropp 🙂

Reply:
🧠